src/components/pokemons.js

```diff
@@ -1,20 +1,25 @@
 import './pokemons.scss';
-import { createElement, appendContent } from '../lib/dom';
+import { createElement } from '../lib/dom';
 
 export function createPokemons(props) {
   const container = createElement('div', {
     className: 'pokemons'
   });
 
-  props.items.forEach(item => {
+  props.items.forEach((item, pokemonId) => {
     const element = createElement('div', {
       innerText: item,
       className: 'pokemon'
     });
+    const pokemonImage = createElement('img', {
+      className: 'pokemonImage',
+      src: `https://pokeres.bastionbot.org/images/pokemon/${pokemonId + 1}.png`
```

Comment:
don't forget the `alt` attribute :)
```suggestion
      src: `https://pokeres.bastionbot.org/images/pokemon/${pokemonId + 1}.png`,
      alt: `image of pokemon #${pokemonId}`
```